v4.5/Makefile

```diff
@@ -0,0 +1,322 @@
+PROFILE := v4.5
```

Comment:
I could probably extract the `CSC`, `CSC_COMMON_ARGS` in a common.make file though.